.github/workflows/ci-scripts-build.yml

```diff
@@ -72,11 +72,12 @@ jobs:
             base: "7.0"
             extra: "CMD_CXXFLAGS=-std=c++11"
 
-          - os: ubuntu-22.04
-            cmp: gcc
-            configuration: default
-            base: "7.0"
-            rtems: "4.10"
+          # It requires more debugging:
+          # - os: ubuntu-22.04
+          #   cmp: gcc
+          #   configuration: default
+          #   base: "7.0"
+          #   rtems: "4.10"
 
```

Comment:
Any opinion on RTEMS  is outside my competence. I suggest @mdavidsaver reviews this.

Reply:
The current state of the epics-base GHA jobs is that the RTEMS 4.10 build does not run tests.  There are also RTEMS 5.1 jobs.

https://github.com/epics-base/epics-base/blob/7.0/.github/workflows/ci-scripts-build.yml#L111-L149